### PR TITLE
[fix] Added total closing in general ledger

### LINKED ISSUE #11893: Add Row in General Ledger Report
Add Row in General Ledger Report to show the closing balance equal to Total Debit - Total Credit or Total Credit - Total Debit.  

Now

screen shot 2017-12-08 at 4 31 34 pm

Before

screen shot 2017-12-08 at 4 32 46 pm

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
**Before**
screen shot 2017-12-12 at 5 04 53 pm

**After**
screen shot 2017-12-12 at 7 09 46 pm

Fixed https://github.com/frappe/erpnext/issues/11893

### CODE REVIEW

File: erpnext/accounts/report/general_ledger/general_ledger.py [hunk 1]
```diff
@@ -193,6 +193,11 @@ def get_data_with_opening_closing(filters, account_details, gl_entries):
 	# closing
 	data.append(totals.closing)
 
+	#total closing
+	total_closing = totals.total_closing
+	total_closing['debit'] = totals.closing.get('debit', 0) - totals.closing.get('credit', 0)
```
Comment (nabinhait) [inline erpnext/accounts/report/general_ledger/general_ledger.py]: It should be set for `debit_in_account_currency` as well

File: erpnext/accounts/report/general_ledger/general_ledger.py [hunk 2]
```diff
@@ -207,7 +212,8 @@ def _get_debit_credit_dict(label):
 	return _dict(
 		opening = _get_debit_credit_dict(_('Opening')),
 		total = _get_debit_credit_dict(_('Total')),
-		closing = _get_debit_credit_dict(_('Closing (Opening + Total)'))
+		closing = _get_debit_credit_dict(_('Closing (Opening + Total)')),
+		total_closing = _get_debit_credit_dict(_('Total Closing (Dr - Cr)'))
```
Comment (nabinhait) [inline erpnext/accounts/report/general_ledger/general_ledger.py]: Let's call it "Closing Balance (Dr - Cr)"

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/accounts/report/general_ledger/general_ledger.py (hunk 2 above)
Comment (rohitwaghchaure) [reply erpnext/accounts/report/general_ledger/general_ledger.py]: Fixed!

File: erpnext/accounts/report/general_ledger/general_ledger.py (hunk 1 above)
Comment (rohitwaghchaure) [reply erpnext/accounts/report/general_ledger/general_ledger.py]: Fixed

**Pull request merged**